package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "start": "NODE_ENV=production ./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
     "start:dev": "nodemon --config nodemon.json",
+    "start:local": "nodemon --ext ts,js,json --exec ts-node ./src/main/server.ts",
```

Comment:
I'm almost certain that `start:dev` does the exactly same thing.

Reply:
reverted file